caffe2/onnx/backend.cc

```diff
@@ -608,15 +608,19 @@ Caffe2Ops Caffe2Backend::CreateGemm(OnnxNode* onnx_node, int opset_version) {
     caffe2::Argument arg_trans_b;
     arg_trans_b.set_name("trans_b");
     arg_trans_b.set_i(trans_b);
-    caffe2::Argument arg_broadcast;
-    arg_broadcast.set_name("broadcast");
-    arg_broadcast.set_i(broadcast);
 
     auto* c2_op = ret.ops.Add();
     BuildOperator(
         c2_op, "MatMul", {input_a, input_b}, {ab}, {arg_trans_a, arg_trans_b});
     c2_op = ret.ops.Add();
-    BuildOperator(c2_op, "Add", {ab, input_c}, {output}, {arg_broadcast});
+    if (opset_version > 6) {
```

Comment:
nit: >= 7 (because it's 7 introduced the broadcast semantics changes)

Reply:
sure